ifrs17-template/Constants/Sharepoint.ipynb

```diff
@@ -37,17 +37,52 @@
         {
             "cell_type": "code",
             "source": [
-                "public class Sharepoint{",
-                "\n    public const string Sites  = \"sites/XXX\"; //Sharepoint sites",
-                "\n    public const string Tenant = \"systemorph.sharepoint.com\";             //Sharepoint tenant",
-                "\n    public const string Root = \"Dev/\";                                    //Dev main directory",
-                "\n    public const string PathToExport = Root + \"ExportFilesTest/\"          //Target export directory ",
-                "\n    public const string PathToImport = Root + \"ImportFiles/\"",
+                "public record SharePointConnectionConfiguration(string Sites = \"sites/XXX\", string Tenant = \"systemorph.sharepoint.com\", string Root = \"Dev/\", bool isApplicationType = false);"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public static void SetDefaults(this Systemorph.SharePoint.SharePointVariable sharePointVariable, SharePointConnectionConfiguration sharePointConnectionConfiguration) {",
+                "\n    if(sharePointConnectionConfiguration.isApplicationType)",
```

Comment:
isApplicationType:  I don't get the logic around it. Markdown should be added.

Reply:
I added the markdown. Please  see the explanations in the previous discussion chain.